### ISSUE TITLE
Mask Annotations can be released?

### ISSUE THREAD

**Issue by sankin97** [Isaaclabe/DGD-Dynamic-3D-Gaussians-Distillation#1]
hi, thank you for your great work. Could you provide Mask Annotation？

**Comment by Isaaclabe** [comment]
Hi, what do you mean by mask annotation? the code to create masks?

**Comment by sankin97** [comment]
Thank you for your quick reply. I am asking about the mask annotation of D-NeRF and HyperNeRF introduced in your paper.

**Comment by Isaaclabe** [comment]
Yes, I will try to realese the code to genereate these masks in the current of the week

**Comment by sankin97** [comment]
Thank you very much for your reply.

**Comment by hxiaoj** [comment]
When to give the mask annotations?

**Comment by Isaaclabe** [comment]
Hello, I am publishing the mask annotation in Mask_annotation.zip file in this main repository. Sorry for the delay..
